backend/dataall/modules/vote/__init__.py

```diff
@@ -1,7 +1,6 @@
 from typing import Set, List, Type
 
 from dataall.base.loader import ModuleInterface, ImportMode
-from dataall.modules.catalog import CatalogApiModuleInterface
 from dataall.modules.vote import api
```

Comment:
this import is unnecessary as its imported in the MI class

Reply:
Agreed!